src/main/resources/io/jenkins/plugins/designlibrary/Layouts/index.jelly

```diff
@@ -0,0 +1,144 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:s="/lib/samples" xmlns:st="jelly:stapler">
+  <s:sample>
+    <div class="jdl-component-sample">
+      <div id="layout-preview" class="jdl-layout-preview jdl-layout-preview--two-column">
+        <div class="jdl-layout-preview__navigation" />
+        <div class="jdl-layout-preview__sidebar">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__content">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__cards">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+        </div>
+      </div>
+    </div>
+
+    <div class="jenkins-tab-pane">
+      <h2 class="jenkins-tab-pane__title">Two column</h2>
+
+      <p class="jdl-leading-paragraph">
+        Two column layouts are for 'hub' style pages, pages that have child pages
+        for the user to navigate to.
+      </p>
+
+      <p class="jdl-paragraph">Examples of such pages include 'Dashboard', 'Project' and 'Plugin Manager'.</p>
+
+      <hr />
+
+      <table class="jdl-dos-donts">
+        <thead>
+          <tr>
+            <th class="jenkins-!-color-success-color">${%do}</th>
+            <th class="jenkins-!-color-error-color">${%dont}</th>
+          </tr>
+        </thead>
+        <tbody>
```

Comment:
Somehow this is a little bit misleading: since your content is placed in two columns (do and don't) when clicking on "Two column" and one column (do) when clicking on "One column" I thought in the beginning that you are talking about splitting the content area in two content columns. But you are actually meaning that you are splitting the screen into a side bar and a single content column.

Reply:
Ah I see what you mean 👍 I've added a `Don't` column to the one-column page so it'll hopefully come across as intended now.